core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -124,6 +124,11 @@
                                    style="margin-left: 10px;">
           </collection-summary-tile>
         </div>
+          <div class="conversation-skin-back-to-collection">
+            <span ng-src="<[collectionSummary.thumbnail_icon_url]>" class="conversation-skin-back-to-collection-header" 
```

Comment:
Use anchor tag to link to collection page. Clicking on the "Back to collection page" text should direct user to the collection page.